lib/src/message.dart

```diff
@@ -12,11 +12,12 @@ import 'body.dart';
 import 'headers.dart';
 import 'http_unmodifiable_map.dart';
 
+/// Retrieves the [Body] contained in the [message].
```

Comment:
Explain that it's a separate function so the message body is accessible within shelf but hidden elsewhere.